src/components/cookie-banner/index.md.njk

```diff
@@ -121,6 +121,13 @@ When the user sets or changes their cookie preferences using the controls on the
 
 ## Research on this component
 
-We remove the visible focus indicator from when shifting focus to the cookie banner. We do this because the cookie banner is the first element on the page and the last thing the user interacted with prior to it being given focus.
+When the user accepts or rejects cookies, a replacement message will display. For example, "Your cookie preferences have been saved." The focus also shifts to this new message.
 
-So we’ve assumed that giving the cookie banner a visible focus indicator in these circumstances would be more likely to confuse users than to help them, especially as the element is not normally operable with a keyboard. However, more research is needed to find out if that assumption is right.
+However, a visible focus indicator does not appear around the replacement message. This is different from [what happens when a user does something in a notification banner](../notification-banner/#reacting-to-something-the-user-has-done).
```

Comment:
> This is different from [what happens when a user does something in a notification banner]

This isn't quite right, because we show the notification banner in response to something the user has done _elsewhere_ – the user is not 'doing something' _in_ the notification banner.

Maybe it's enough to say 'This is different to the notification banner, where we do show a visible focus indicator'?

Reply:
```suggestion
However, a visible focus indicator does not display around the replacement message. This different from the notification banner, which does display a visible focus indicator.
```